CHANGELOG.md

```diff
@@ -5,34 +5,34 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+- New sample scene: **Boss cam** sample scene demonstrates a camera setup to follow the player and to look at the player and the boss. The scene provides  examples of custom extensions.
 
 
 ## [2.8.0-exp.1] - 2021-03-31
-- Added simplified modes for impulse generation, added secondary reaction settings to Impulse Listener.
+- Added simplified modes for impulse generation and added secondary reaction settings to Impulse Listener.
 - Added Storyboard support for ScreenSpaceOverlay and ScreenSpaceCamera camera render modes.
 - Added DampingIntoCollision and DampingFromCollision properties to Cinemachine3rdPersonFollow to control how gradually the camera moves to correct for occlusions.
 - Added CinemachineCore.OnTargetObjectWarped() to warp all vcams targeting an object.
 - Added ability for vcam to have a negative near clip plane.
-- Added Draggable Game Window Guides toggle in Cinemachine preferences. If disabled, Game Window guides are only for visualization.
-- Added button to virtual camera inspectors to auto-generate CinemachineInputProvider component if missing.
+- Added Draggable Game Window Guides toggle in Cinemachine preferences. When disabled, the Game Window guides only provide visualization.
+- Added button to virtual camera inspectors to auto-generate CinemachineInputProvider component when the component is missing.
 - Default PostProcessing profile priority is now configurable and defaults to 1000.
-- Cinemachine3rdPersonFollow can operate without the physics module, and without collision resolution.
-- Bugfix: 3rdPersonFollow collision resolution was failing when the camera radius was large.
-- Bugfix: 3rdPersonFollow damping was being done in world space instead of camera space.
-- Bugfix: 3rdPersonFollow was stuttering when Z damping was high.
-- Regression fix: CinemachineInputProvider had stopped providing input.
-- Bugfix: Lens aspect and sensorSize were not getting updated if lens OverrideMode != None.
-- Bugfix: Changing targets on a live vcam was misbehaving.
-- Bugfix: Framing transposer now handles empty groups.
+- Cinemachine3rdPersonFollow can operate without the physics module and without collision resolution.
+- Bugfix: 3rdPersonFollow collision resolution failed when the camera radius was large.
+- Bugfix: 3rdPersonFollow damping operated in world space instead of camera space.
+- Bugfix: 3rdPersonFollow stuttered when Z damping was set high.
+- Regression fix: CinemachineInputProvider stopped providing input.
+- Bugfix: Lens aspect and sensorSize were not updated when lens OverrideMode != None.
+- Bugfix: Changing targets on a live vcam misbehaved.
+- Bugfix: Framing transposer did not handle empty groups.
 - Bugfix: Interrupting a transition with InheritPosition enabled was broken.
-- Bugfix: Cinemachine3rdPersonFollow was not handling collision by default.
```

Comment:
Bugfix: Cinemachine3rdPersonFollow was handling collision by default, now it is disabled by default.

Reply:
Done.